docusaurus/sidebars.js

```diff
@@ -16,7 +16,7 @@ const sidebars = {
   docsSidebar: [
     {
       type: 'html',
-      value: 'LEARN', // The HTML to be rendered
+      value: '<a href="/docs/learn/introduction/">LEARN', // The HTML to be rendered
```

Comment:
this is improper html. this 'works'?

Reply:
If you're going with the `<a>` approach -- i dunno if we should, you need to style the link to look the same. notice the REFERENCE difference

![image](https://user-images.githubusercontent.com/46322585/214945075-c144b71b-04e4-41b9-bd22-07e3b2dee674.png)

Reply:
Forgot the closing tag.